appsettings.json

```diff
@@ -6,7 +6,7 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "mongodb://localhost:27017"
+    "DefaultConnection": "Server=::1;Port=5432;Database=shipping_service_db;User Id=admin;Password=password"
```

Comment:
Had this one in my config.
Trusted_Connection=True; MultipleActiveResultSets=True
Any use of them?

Reply:
I think they are specific to SQL Server.

Reply:
Probably. Resolved then.